src/main.rs

```diff
@@ -11,23 +11,23 @@ use orchestrator::Runner;
 fn cmd_from_string(s: String) -> Result<std::process::Command, &'static str> {
     let mut iter = s.split_ascii_whitespace();
     println!("{:?}\n", iter);
-   
+
     let cmd;
     let first;
 
     first = iter.nth(0);
     match first {
         Some(c) => {
-            cmd = c;            
+            cmd = c;
         }
         None => {
             // Not a great err msg...
             return Err("Expected there to be at least one thing in the command");
         }
     }
 
-    let base_command = Command::new(cmd);
-    let command = base_command.args(iter);
+    let mut command = Command::new(cmd);
+    command.args(iter);
```

Comment:
This returns the new command with those args applied, so _this_ is the thing we need to return. =\

Reply:
I haven't tried running it yet, but I'm guessing that `command.args(iter);` will apply the args to `command` defined above which we finally return. That's just a guess though lol. Will have to run and check it.